### PR TITLE
Update github to tokenutil

### PR DESCRIPTION
**Pull request by jefferai**

### CODE REVIEW

File: builtin/credential/github/path_config.go [hunk 1]
```diff
@@ -51,48 +49,77 @@ API-compatible authentication server.`,
 			logical.ReadOperation:   b.pathConfigRead,
 		},
 	}
+
+	tokenutil.AddTokenFields(p.Fields)
+	p.Fields["token_policies"].Description += ". This will apply to all tokens generated by this auth method, in addition to any configured for specific users/groups."
```
Comment (vishalnayak) [inline builtin/credential/github/path_config.go]: Please paraphrase "any configured"

File: builtin/credential/github/path_config.go [hunk 2]
```diff
@@ -109,23 +136,26 @@ func (b *backend) pathConfigRead(ctx context.Context, req *logical.Request, data
...
+	config.PopulateTokenData(d)
 
-	resp := &logical.Response{
-		Data: map[string]interface{}{
-			"organization": config.Organization,
-			"base_url":     config.BaseURL,
-			"ttl":          config.TTL,
-			"max_ttl":      config.MaxTTL,
-		},
+	if config.TTL > 0 {
```
Comment (vishalnayak) [inline builtin/credential/github/path_config.go]: Can we add a comment here (like the other PR that this is for BC) and a warning in the response?

File: builtin/credential/github/path_config.go (hunk 1 above)
Comment (jefferai) [reply builtin/credential/github/path_config.go]: Done

File: builtin/credential/github/path_config.go (hunk 2 above)
Comment (jefferai) [reply builtin/credential/github/path_config.go]: I'll add the comment. Chris didn't want to change from 204 to 200 so path-help and description in the UI will have to suffice.

**Pull request merged**